add-on/src/popup/quick-upload.js

```diff
@@ -47,23 +71,37 @@ function quickUploadStore (state, emitter) {
 
 function quickUploadPage (state, emit) {
   const onFileInputChange = (e) => emit('fileInputChange', e)
-
+  const {filesCount, peerCount} = state
   return html`
-    <div class="helvetica white pt3" style="background: linear-gradient(to bottom, #041727 0%,#043b55 100%); height:100%;">
-      <div class="measure-wide center pa3 tc">
-        ${logo({
-          size: 100,
-          path: '../../icons',
-          heartbeat: false
-        })}
-        <h1 id="quickUploadMessage" class='mb1'>
-          ${browser.i18n.getMessage('panel_quickUpload')}
-        </h1>
-        <p id="form">
-          <input type="file" id="quickUploadInput" onchange=${onFileInputChange} />
-          <br>
-          ${state.message}
-        </p>
+    <div class="avenir pt5" style="background: linear-gradient(to top, #041727 0%,#043b55 100%); height:100%;">
+      <div class="mw8 center pa3 white">
+        <header class="flex items-center">
+          ${logo({
+            size: 80,
+            path: '../../icons',
+            heartbeat: false
+          })}
+          <div class="pl3">
+            <h1 class="f2 fw5 ma0">${browser.i18n.getMessage('panel_quickUpload')}</h1>
+            <p class="f3 fw2 lh-copy ma0 light-gray">Currently sharing <strong class="fw6">${filesCount}</strong> files with <strong strong class="fw6">${peerCount}</strong> IPFS peers</p>
+          </div>
+        </header>
+
+        <label for="quickUploadInput" class='db relative mv5 hover-inner-shadow' style="border:solid 2px #6ACAD1">
+          <input class="db absolute pointer w-100 h-100 top-0 o-0" type="file" id="quickUploadInput" onchange=${onFileInputChange} style=""/>
+          <div class='dt dim' style='padding-left: 100px; height: 300px'>
+            <div class='dtc v-mid'>
+              <span class="f3 link dim br1 ph4 pv3 dib white" style="background: #6ACAD1">
+                Pick a file
```

Comment:
I feel adding i18 keys (locales) should be a part of this PR – otherwise we may miss it and make translators super sad 😿

Reply:
Good point. Sorry, that was lazy of me!

Reply:
_thinking out loud_ I think I generally hold off on i18n until I've got some general approval on a new design.

Reply:
Now that you mentioned, I got your point: it may be counter-productive to translate too early (eg. we may change/remove labels and translation work is lost). Especially now that we have multiple people working on multiple things. 

Perhaps we could make the process more efficient by agreeing that _it is okay to commit english inline in places that can change before stable release_, and instead add "localization" step before final non-beta release?